### DISCUSSION TITLE
Gutenburg Update Caused My WooCommerce Pages to Appear Blank

### DISCUSSION THREAD

**Discussion by diannemize** [WordPress/gutenberg#50317] [Q&A]
I have had to deactivate Gutenburg because an update around mid-April caused my WooCommerce pages to appear blank.  Does anyone have a solution?

**Comment by t-hamano** [comment]
Hi @diannemize,

Which page of the WooCommerce plugin was blanked out? It would be great if you could attach the steps to access that page or a screenshot.

Also, you might want to report this issue in [the support forum](https://wordpress.org/support/plugin/woocommerce/) or on [GitHub](https://github.com/woocommerce/woocommerce).
